appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -130,31 +126,31 @@ internal fun Modifier.styleBorder(
     }
 )
 
-internal fun Modifier.styleSize(style: ComponentStyle, noSizeFillMax: Boolean = false) = this.then(
+internal fun Modifier.styleSize(style: ComponentStyle, contentMode: ComponentContentMode = ComponentContentMode.FIT) = this.then(
```

Comment:
most things _dont_ stretch to fill size - so default here is FIT, but images can optionally pass in FILL.